assets/airports/cyhz.json

```diff
@@ -0,0 +1,660 @@
+                "14": ["KEBOT", ["VIMLA", "A50|S210"], ["IGVOP", "A40"]],
+                "32": [["MENVO", "A50|S230"], ["GABAP", "A40|S210"], "KIXIT"],
+                "05": ["OVTUV", ["PEPTA", "A50"], ["OBNOM", "A40"]],
+                "23": [["LEROS", "A30|S200"]]
+            },
+            "draw": [
+                ["CAYLY*", "BOSUB", "KEBOT", "VIMLA", "IGVOP"],
+                ["BOSUB", "MENVO", "GABAP", "KIXIT"],
+                ["BOSUB", "OVTUV", "PEPTA", "OBNOM"],
+                ["BOSUB", "LEROS"]
+            ]
+        },
+        "FUNDY1": {
+            "icao": "FUNDY1",
+            "name": "Fundy One",
+            "entryPoints": {
+                "EBONY": ["EBONY", "EBKID"],
+                "ALLEX": ["ALLEX", "BAFDY"]
+            },
+            "body": ["MUVOX"],
+            "rwy": {
+                "14": [["TETAR", "A30|S200"]],
+                "32": [["EBLUV", "A50|S210"], ["EMBAX", "A40"]],
+                "05": [["ODKAS", "A30|S200"]],
+                "23": [["LOGPO", "A50|S210"], ["DUTIR", "A40"], ["LEROS", "A30"]]
+            },
+            "draw": [
+                ["EBONY*", "EBKID", "MUVOX", "TETAR"],
+                ["ALLEX*", "BAFDY", "MUVOX", "EBLUV", "EMBAX"],
+                ["MUVOX", "ODKAS"],
+                ["MUVOX", "LOGPO", "DUTIR", "LEROS"]
+            ]
+        },
+        "LIRLA1": {
+            "icao": "LIRLA1",
+            "name": "Lirla One",
+            "entryPoints": {
+                "AGMIR": [
+                    "AGMIR"
+                ]
+            },
+            "body": ["LIRLA"],
+            "rwy": {
+                "14": [["VIMLA", "A50|S210"], ["IGVOP", "A40"]],
+                "32": [["VOKIL", "A30"]],
+                "05": [["PEPTA", "A50|S210"], ["OBNOM", "A40"]],
+                "23": [["LEROS", "A31|S200"]]
+            },
+            "draw": [
+                ["AGMIR", "LIRLA", "VIMLA", "IGVOP"],
+                ["LIRLA", "VOKIL"],
+                ["LIRLA", "PEPTA", "OBNOM"],
+                ["LIRLA", "LEROS"]
+            ]
+        },
+        "PEGGY1": {
+            "icao": "PEGGY1",
+            "name": "Peggy One",
+            "entryPoints": {
+                "TUSKY": [
+                    "TUSKY"
+                ],
+                "ELERI": [
+                    "ELERI"
+                ]
+            },
+            "body": ["ETGAR"],
+            "rwy": {
+                "14": [["ILOTA", "A50|S210"], ["MODIR", "A40"], "DUNUB"],
+                "32": [["NOTUB", "A50|S230"], ["EMBAX", "A40|S210"], "EBLIT"],
+                "05": [["ODKAS", "A30|S200"]],
+                "23": ["OBNOM", ["RUDBU", "A50|S210"], ["OVTUV", "A40"]]
+            },
+            "draw": [
+                ["TUSKY", "ETGAR", "ILOTA", "MODIR", "DUNUB"],
+                ["ETGAR", "NOTUB", "EMBAX", "EBLIT"],
+                ["ETGAR", "ODKAS"],
+                ["ETGAR", "OBNOM", "RUDBU", "OVTUV"]
+            ]
+        }
+    },
+    "spawnPatterns": [
```

Comment:
From [Wikipedia](https://en.wikipedia.org/wiki/Halifax_Stanfield_International_Airport), the 2016 stats indicate 80,278 aircraft movements. IAW the [traffic level specs](https://github.com/openscope/openscope/blob/develop/documentation/airport-file-standards.md), this should fall between 9acph and 28acph. Currently, there's a total of 9.96acph.

This is good to go in that regard. :+1: